contrib/kubernetes/flowlogs-pipeline.conf.yaml

```diff
@@ -71,78 +90,156 @@ parameters:
       - input: dstIP
         output: dstLocation
         type: add_location
-    type: network
-- extract:
+- name: extract_conntrack
```

Comment:
do you know if this is going to be compatible with a "direct prometheus" pipeline, ie. without Aggregation stage?
Said differently, if a prom-encode is wired directly after extract_conntrack, will it work? (granted we include fixes from https://github.com/netobserv/flowlogs-pipeline/pull/266 )
I guess there will be some adjustments to do so that prom-encode picks only "flowLog" output, and ignore "new/endConnection"

Reply:
> do you know if this is going to be compatible with a "direct prometheus" pipeline, ie. without Aggregation stage?
I can't think why not, so I would say yes. I think it should work.
Instead of adding adjustments to prom-encode to pick only "flowLog" records, we can add a `Transform` step between `extract_conntrack` and `encode_prom` to filter the records.